ubyssey/helpers.py

```diff
@@ -30,14 +29,14 @@ def get_frontpage(reading_times=None, section=None, section_id=None, sections=[]
         context.update(reading_times)
 
         query = """
-        SELECT *,
-            TIMESTAMPDIFF(SECOND, published_at, NOW()) as age,
+            SELECT *, TIMESTAMPDIFF(SECOND, published_at, NOW()) as age,
             CASE reading_time
                  WHEN 'morning' THEN IF( CURTIME() < %(morning_start)s, 1, 0 )
                  WHEN 'midday'  THEN IF( CURTIME() >= %(midday_start)s AND CURTIME() < %(midday_end)s, 1, 0 )
                  WHEN 'evening' THEN IF( CURTIME() >= %(evening_start)s, 1, 0 )
                  ELSE 0.5
-            END as reading
+            END as reading,
+            TIMESTAMPDIFF(DAY, published_at, NOW()) <= 14 as age_deadline
```

Comment:
Can we replace `14` with a named parameter (like `%(age_deadline)s`) and then make the age deadline an argument for `get_frontpage`? It'd be nice to make it a bit more flexible.